### ISSUE TITLE
track version in ~/.solana-workbench, or wherever

### ISSUE THREAD

**Issue by nathanleclaire** [workbenchapp/solana-workbench#133]
i think we've run into enough issues now w/ breaking changes between versions that it might be worth keeping track of what version(s) have been there before so we can at least detect / error out when things don't make sense

**Comment by SvenDowideit** [comment]
yup, it'd also be a good idea to list the issues you're talking about
